src/backend/storeManagers/nile/setup.ts

```diff
@@ -93,7 +93,7 @@ export default async function setup(
     const exeArguments = action.Args ?? []
 
     if (isWindows) {
-      const command = ['Start-Process', '-FilePath', action.Command]
+      const command = ['-NoProfile','Start-Process', '-FilePath', action.Command]
```

Comment:
```suggestion
      const command = ['-NoProfile', 'Start-Process', '-FilePath', action.Command]
```

I suggest setting up your code editor to format files on save, so these details are solved automatically